jira/client.py

```diff
@@ -1016,8 +1006,9 @@ def file_stream() -> MultipartEncoder:
         )
         if jira_attachment.size == 0:
             raise JIRAError(
-                "Added empty attachment via %s method?!: r: %s\nattachment: %s"
-                % (method, r, jira_attachment)
+                "Added empty attachment?!: r: {}\nattachment: {}".format(
```

Comment:
Looks like pyupgrade doesn't always convert to an f-string:

https://github.com/asottile/pyupgrade#f-strings

> note: pyupgrade is intentionally timid and will not create an f-string if it would make the expression longer or if the substitution parameters are sufficiently complicated (as this can decrease readability).